packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -106,7 +106,6 @@ We'll expose a `BottomSheet` component with the following API:
 | `leading` | `React.ReactNode` | `undefined` | leading asset or icon to be placed at the left most side of the BottomSheetHeader |          |
 | `trailing` | `Badge, Link, Text, IconButton`, `undefined` | undefined  |     trailing component to be placed at the right most side of the BottomSheetHeader     |    |
 | `titleSuffix` | `Counter` | `undefined` | A component to be placed adjacent to the title text |          |
-| `showDivider` | `boolean` | `true` | Show or hide the divider |          |
```

Comment:
Last minute change: Rastogi & Abhishek discussed that it's best if we don't expose this prop to consumers.